<launch>

	<!-- Low level system nodes (running at embedded host) -->
	<node
		pkg="lrm_drivers"
		name="roboteq"
		type="roboteq_node"
	>
		<param
			name="port"
			value="/dev/ttyS0" />
		<param
			name="model"
			value="AX2550" />
		<param
			name="steering_angle_ratio"
			value="4.1" />
		<param
			name="publish_encoders"
			value="true" />
	</node>

	<node
		pkg="lrm_drivers"
		name="arduino"
		type="throttle_node"
	>
		<param
			name="port"
			value="/dev/ttyUSB0" />
		<param
			name="max_value"
			value="20" />

		<remap
			from="throttle_commands"
			to="velocity_commands" />

	</node>

	<include file="$(find lrm_drivers)/launch/joy_teleop.launch" />


</launch>
